src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -230,6 +229,8 @@ std::vector<ov::AnyMap> OVPropertiesTestsWithCompileModelProps::getROOptionalPro
         res.push_back({{ov::PropertyName(ov::device::type.name(), ov::device::type.mutability), nullptr}});
         res.push_back({{ov::PropertyName(ov::execution_devices.name(), ov::execution_devices.mutability), nullptr}});
         res.push_back({{ov::PropertyName(ov::available_devices.name(), ov::available_devices.mutability), nullptr}});
+        res.push_back(
```

Comment:
please move under the condition if (!is_sw_device) this property in the function OVPropertiesTestsWithCompileModelProps::getRWMandatoryPropertiesValues [str 271](https://github.com/openvinotoolkit/openvino/blob/2d22dd70c0f1fa783fa6c34a5d67b009396a7bc1/src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp#L271)